crates/cext/src/circuit.rs

```diff
@@ -68,6 +68,29 @@ pub extern "C" fn qk_circuit_new(num_qubits: u32, num_clbits: u32) -> *mut Circu
     Box::into_raw(Box::new(circuit))
 }
 
+/// @ingroup QkCircuit
+/// Create a copy of a QkCircuit.
+///
+/// @param circuit A pointer to the circuit to copy
+///
+/// @return A new pointer to a copy of the input `circuit`.
```

Comment:
The only nit I could find (can't leave a review without any comment! 😛)
```suggestion
/// Create a copy of a ``QkCircuit``.
///
/// @param circuit A pointer to the circuit to copy.
///
/// @return A new pointer to a copy of the input ``circuit``.
```